extension/icu/third_party/icu/common/unicode/localpointer.h

```diff
@@ -43,6 +43,7 @@
 #if U_SHOW_CPLUSPLUS_API
 
 #include <memory>
+#include "duckdb/common/unique_ptr.hpp"
```

Comment:
Lets try to avoid modifying the ICU library itself (i.e. the code in third_party)